Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -186,11 +195,17 @@ public enum TypeDescription: Codable, Hashable {
     case let .dictionary(key, value):
       try container.encode(key, forKey: .dictionaryKey)
       try container.encode(value, forKey: .dictionaryValue)
+    case let .closure(arguments, doesThrow, returnType):
+      try container.encode(arguments, forKey: .typeDescriptions)
+      if doesThrow {
+        try container.encode("throws", forKey: .text)
+      }
+      try container.encode(returnType, forKey: .typeDescription)
     }
   }
 
   enum CodingKeys: String, CodingKey {
-    /// The value for this key is the case encoded as a string.
+    /// The value for this key is the case encoded as a String.
```

Comment:
This line change is unrelated to the rest of the PR but figured I'd make the capitalization consistent while I was here.